platform-includes/getting-started-verify/javascript.nestjs.mdx

```diff
@@ -1,14 +1,6 @@
 ```javascript {"onboardingOptions": {"performance": "1-4, 12"}}
-Sentry.startSpan({
-  op: "test",
-  name: "My First Test Span",
-}, () => {
-  setTimeout(() => {
-    try {
-      foo();
-    } catch (e) {
-      Sentry.captureException(e);
-    }
-  }, 99);
-});
+@Get("/debug-sentry")
```

Comment:
Does `@Get(...)` create a span, i.e. is it relevant to performance tracing? If so, you should update the `onboardingOptions` to remove line 1. If it's not relevant, then just remove the onboarding options from the snippet.